### PR TITLE
implement a build script

### PR DESCRIPTION
**Pull request by VoR0220**
Signed-off-by: RJ Catalano <rj@monax.io>

### CODE REVIEW

File: scripts/build.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+fi
+
+cd $(dirname "$0")
+mkdir -p build
+cd build
+cmake .. -DCMAKE_BUILD_TYPE="$BUILD_TYPE"
+make -j2
+
+if [ -z $CI ]; then
+	install solc/solc /usr/local/bin && install test/soltest /usr/local/bin
```
Comment (axic) [inline scripts/build.sh:16]: Should this be part of this script?

File: docs/installing-solidity.rst [hunk 2]
```diff
@@ -251,4 +257,4 @@ Example:
 3. a breaking change is introduced - version is bumped to 0.5.0
 4. the 0.5.0 release is made
 
-This behaviour works well with the version pragma.
+This behaviour works well with the version pragma.
```
Comment (axic) [inline docs/installing-solidity.rst:261]: What changed here?

File: scripts/build.sh (hunk 1 above)
Comment (VoR0220) [reply scripts/build.sh:16]: Yes. This is a typical pain that I and many others have run into on OS X where you expect it to automatically link but it doesn't.

File: docs/installing-solidity.rst (hunk 2 above)
Comment (VoR0220) [reply docs/installing-solidity.rst:261]: ...I think it got pushed down? I honestly have no idea why...

File: scripts/build.sh (hunk 1 above)
Comment (axic) [reply scripts/build.sh:16]: But this changes behaviour for Travis at the moment as I do no see $CI being set by Travis?

Comment (VoR0220) [reply scripts/build.sh:16]: It's actually default set by Travis :) https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables

Comment (axic) [reply scripts/build.sh:16]: Cool, didn't know that. If it installs the binary after compiling that should be mentioned in the documentation though. I guess I personally won't be using the script if it installs 😉

Comment (VoR0220) [reply scripts/build.sh:16]: Agreed. Ill fix.

Comment (VoR0220) [reply scripts/build.sh:16]: @axic fixed!

**Review by axic — APPROVED** [review]

**Pull request merged**